src/drivers/distance_sensor/tfmini/TFMINI.cpp

```diff
@@ -75,14 +75,14 @@ TFMINI::init()
 	int32_t hw_model = 1; // only one model so far...
 
 	switch (hw_model) {
-	case 1: // TFMINI (12m, 100 Hz)
+	case 1: // TF LiDAR Series (8m to 350m depending upon the product model, 100 Hz)
```

Comment:
Please keep this the same for existing tfmini users, unless it's possible to detect from the sensor itself.

Reply:
It will not affect TFmini usage. I just changed the range and set it to the LiDAR having maximum range TF350. The protocol of all these LiDARs is same. I am working in this company I have tested mini after changing. I have just generalized the term. Later I will request to make changes in the documentation to add more details about every LiDAR so that users are not confused.
By the way most of the people mix mini and mini-S they say they have mini but when I asked them for serial number its mini-S.